web/src/components/taxonomies/taxonomies-tree.tsx

```diff
@@ -53,23 +55,27 @@ export const TaxonomiesTree: FC<TaxonomiesTreeProps> = ({
         [expandNode]
     );
 
-    const handleSelect = useCallback((selectedKeys, info) => {
-        setSelectedKeys(selectedKeys);
-        if (selectedAnnotation?.labels) {
-            const labelToChangeIdx: number = selectedAnnotation?.labels.findIndex(
-                (label: AnnotationLabel) => label.annotationId === selectedAnnotation.id
-            );
+    const handleSelect = useCallback(
+        (handleSelectedKeys, info) => {
```

Comment:
let's rename handleSelectedKeys to selectedNodes for example, as it's not a function